src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/RampedHaplotypeCallerEngine.java

```diff
@@ -145,7 +145,7 @@ private void tearRamps() {
             throw new RuntimeException(e);
         }
     }
-    private class CallRegionContext {
```

Comment:
why?

Reply:
Because it had a squiggle in IntelliJ? Always static all the time.